common/service/config/pprof.go

```diff
@@ -53,10 +63,11 @@ func (initializer *PProfInitializerImpl) Start() error {
 		return nil
 	}
 
-	go func() {
-		initializer.Logger.Info("PProf listen on %d", port)
-		http.ListenAndServe(fmt.Sprintf("localhost:%d", port), nil)
-	}()
-
+	if atomic.CompareAndSwapInt32(&pprofStatus, pprofNotInitialized, pprofInitialized) {
```

Comment:
What is the situation where Start can be called multiple times?

Reply:
./cadence start, i.e. local run